documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1770,6 +1770,14 @@ Emitting the tombstone event (the default behavior) allows Kafka to completely d
 |_n/a_
 |An optional comma-separated list of regular expressions that match the fully-qualified names of character-based columns whose values should be replaced in the change event message values with a field value consisting of the specified number of asterisk (`*`) characters. Multiple properties with different lengths can be used in a single configuration, although in each the length must be a positive integer or zero. Fully-qualified names for columns are of the form _databaseName_._tableName_._columnName_, or _databaseName_._schemaName_._tableName_._columnName_.
 
+|`column.mask.hash._hashAlgorithm_.with.salt._salt_`
+|_n/a_
+|An optional comma-separated list of regular expressions that match the fully-qualified names of character-based columns whose values should be pseudonyms in the change event message values with a field value consisting of the hashed value using the algorithm `_hashAlgorithm_` and salt `_salt_`.
+Based on the used hash function referential integrity is kept while data is pseudonymized. Supported hash functions are described in the https://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#MessageDigest[MessageDigest section] of the Java Cryptography Architecture Standard Algorithm Name Documentation.
```

Comment:
The Oracle base URL should go into some external var (it's used in other places, too).

@Naros, what's the right place, _debezium.github.io/playbook.yml_?

Reply:
I don't see base URLs being treated differently elsewhere. What do you propose?

Reply:
@JanHendrikDolling In general, if a link is used in more than one place, ideally we want to centralize it as it makes updating that link in the future much easier.  Furthermore, our docs are in a bit of a WIP phase as they're being modularized into more reusable pieces, see the docs for MySQL as an example.

That said, take a look at `documentation/modules/antora.yml`.

This is the component descriptor that Antora uses when it builds our documentation.  In this file there is an `asciidoc > attributes` section where you can add an entry for this link and update all impacted files where this explicit link it used.  Personally I would probably add the entry as follows:
```
`link-java7-standard-names` : 'https://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#MessageDigest'
```